tests/unit/config/config_test.py

```diff
@@ -1138,7 +1138,21 @@ def test_expand_path_with_tilde(self):
         self.assertEqual(result, user_path + 'otherdir/somefile')
 
 
-@pytest.mark.xfail(IS_WINDOWS_PLATFORM, reason='paths use slash')
```

Comment:
Dammit, I fucked up the rebase and didn't spot this.

I know we don't merge on red build, I had assumed it was something else.